### PR TITLE
Add ffpyplayer package.

### PR DESCRIPTION
**Pull request by matham**

### CODE REVIEW

**Comment by matham** [comment]
The error seems to be that "No files or script found for output ffpyplayer". It builds and installs the package successfully, but then doesn't generate a output package. I didn't overwrite the output as far as I can see, so I'm not sure why it doesn't see the generated files?

**Comment by matham** [comment]
OK, will fix the example.

Regarding the install command, I used the default initially, but then
switched to the current one because I thought that may help with the error.
But it didn't.

**Comment by matham** [comment]
I fixed the things you mentioned. From the error message it seemed that it was installing to host rather than build. I explicitly made it run `$PREFIX/python` because that should be the build python, but it seems to run it in host python. I guess the issues is with host/build confusion although I'm not sure why.

**Comment by matham** [comment]
Alright, it's all working now on all OS's. The problem was that I had to declare python as a dep so that I could then use `$PYTHON` in the build. And then on windows I was missing stdinttypes, which I fixed by adding the conda dep. The only remaining issue was on win+2.7, so I excluded py2.7, since I don't plan to maintain 2.7 support anyway for long.

Should be ready to merge now.

File: recipes/ffpyplayer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py < 27 or win and py < 35]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python
```
Comment (jjhelmus) [inline recipes/ffpyplayer/meta.yaml]: python and all the other requirements should be in the `host` section.

File: recipes/ffpyplayer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    - cython
+    - sdl2
+    - sdl2_mixer
+    - ffmpeg
+    - libflac  # [win]
+    - libvorbis  # [win]
+    - pkgconfig
+    - msinttypes  # [win]
+  host:
+    - {{ compiler('c') }}
```
Comment (jjhelmus) [inline recipes/ffpyplayer/meta.yaml]: remove `{{ compiler('c') }}` from the host section

File: recipes/ffpyplayer/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    - python
+    - sdl2
+    - sdl2_mixer
+    - ffmpeg
+    - libflac  # [win]
+    - libvorbis  # [win]
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
```
Comment (jjhelmus) [inline recipes/ffpyplayer/meta.yaml]: These commends should be removed.

**Comment by matham** [comment]
I fixed the requested changes.

**Comment by matham** [comment]
@conda-forge/staged-recipes the PR should be mergable, can you please take a look?

**Pull request merged**